tests/versioning_test.go

```diff
@@ -1392,6 +1393,76 @@ func (s *versioningIntegSuite) dispatchCron() {
 	s.GreaterOrEqual(runs2.Load(), int32(3))
 }
 
+func (s *versioningIntegSuite) TestDescribeTaskQueue() {
+	// force one partition
```

Comment:
Curious why this is required, can you clarify in a comment in code?

Reply:
DescribeTaskQueue only describes the root (or you can give it the internal name of a partition)